nc-news/src/App.css

```diff
@@ -74,6 +74,24 @@ background-color:floralwhite;
   list-style-type: none;
 }
 
+.commentList {
+  list-style-type: none;
+  border-style: double;
+  text-size-adjust:inherit
+}
+
+#commentUser {
+  text-align: left;
+}
+
+#commentDate {
+text-align: left;
+}
+
+#commentVote{
+  text-align: right;
+  padding:1rem;
+}
```

Comment:
These id's should be swapped for classes. There's lots of comments on the page and id's should be unique.